### PR TITLE
Convert to node

### PR DESCRIPTION
**Pull request by krogers78**
Converting the project to Node and updating the HTML and Javascript to reflect skills learned in the degree.

### CODE REVIEW

**Comment by LevanceWam** [comment]
Khloe I went through your code and I didn't see any issues with it everything looks really clean and the readme is very easy to understand and follow you look good to me I will cloase this around 1030 great job again.

**Comment by drubiodev** [comment]
LGTM

File: src/assets/js/battle.js [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+  }
+  // Function to handle a dead player
+  const deadPlayer = (play) => {
+    document.querySelector('#atkMessage').innerHTML = `${play.name} has died`;
+    document.querySelector('#atkBtn').style.display = 'none';
+    document.querySelector('#fleeBtn').style.display = 'none';
+  };
+  // Load the page with the initial player data
+  const initialLoad = (play, gob, el, gi, ra) => {
+    console.log('Player', play);
```
Comment (MikeSewell) [inline src/assets/js/battle.js]: I know this is on the front-end, so try to only use logs when you need the information.

File: src/assets/js/battle.js [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+    gob.currentHP <= 0 ? enemyDecider(play, gob, el, gi, ra) : null;
+    el.currentHP <= 0 ? enemyDecider(play, gob, el, gi, ra) : null;
+    gi.currentHP <= 0 ? enemyDecider(play, gob, el, gi, ra) : null;
+    ra.currentHP <= 0 ? enemyDecider(play, gob, el, gi, ra) : null;
+    play.currentHP <= 0 ? deadPlayer(play) : null;
+    console.log(currentEnemy);
+  };
+  // Function to determine the attack strength
+  const attackAmount = (play, enemy) => {
+    // Random number and DOM element
```
Comment (MikeSewell) [inline src/assets/js/battle.js]: Use meaningful comments, adding unneeded comments can make the code a pain to read. [link to info on comments in javascript](https://www.w3.org/wiki/JavaScript_best_practices#Comment_as_much_as_needed_but_not_more)

File: src/assets/js/classes/character.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  decreaseGold() {
+    this.gold -= 50;
+    return this.gold;
+  }
+  characterInfo() {
+    console.log(`Name: ${this.name} Max HP: ${this.maxHP} Current HP: ${
+      this.currentHP
+    } Weapon: ${this.weapon.weaponName} Defense: ${this.defense}`);
+  }
+  // characterInfoAlert() {
```
Comment (MikeSewell) [inline src/assets/js/classes/character.js]: This goes with my last comment.

File: src/assets/js/battle.js (hunk 1 above)
Comment (krogers78) [reply src/assets/js/battle.js]: I'll definitely go through and remove those.  I forgot they were actually there honestly.

File: src/assets/js/battle.js (hunk 2 above)
Comment (krogers78) [reply src/assets/js/battle.js]: I'll definitely take a look at that. Thanks!

**Comment by drubiodev** [comment]
looks good to me @MikeSewell

**Pull request merged**